atlasdb-cassandra-integration-tests/src/test/java/com/palantir/atlasdb/keyvalue/cassandra/SchemaMutationLockIntegrationTest.java

```diff
@@ -119,15 +123,35 @@ public void shouldWrapCheckedExceptionsInARuntimeException() {
     }
 
     @Test
-    public void testLocksTimeout() throws InterruptedException, ExecutionException, TimeoutException {
```

Comment:
I think we discussed offline and decided we want to bring this back but with a pretty long timeout because large schema mutations can take a long time.